lib/xml_util.rb

```diff
@@ -5,6 +5,10 @@ def self.translatable_nodes(xml)
     xml.xpath('//content:text[@i18n-id]')
   end
 
+  def self.translatable_node_attrs(xml)
+    xml.xpath("//@*[contains(name(),'-i18n-id')]")
```

Comment:
+1 I knew there was a way to do this with XPath, I just hadn't made it quite this far yet ;)